### DISCUSSION TITLE
ToSyncList is causing unwanted behavior

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3450] [Questions]
I am sure the solution is simple : ) We have an application that has a combox with data source of Csla readonly collection.
We had filtered the collection using ToSyncList butdon't want LinqObservableCollection anymore.

ToList(ourchildobject) or ToList() gives us System.Collections.Generic.List<ourchildobject>

We would like to have 'our' Csla collection as return value.

Kind regards

**Comment by rockfordlhotka** [comment]
I'm not sure I follow?

You are (I assume) using the `Source` property to get back to the original list? And that property returns an `IList<T>`.

The `LinqObservableCollection` type doesn't maintain the original collection type, but you can cast it back to that type if you know it in your code.
